Avoid using expat's m4 files

Upstream-Status: Inappropriate [Other]
Workaround specific to our build system.

Signed-off-by: Kevin Tian <kevin.tian@intel.com>
Signed-off-by: Scott Garman <scott.a.garman@intel.com>

Updated to apply over expat 2.1.0

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>

Index: expat-2.2.5/configure.ac
===================================================================
--- expat-2.2.5.orig/configure.ac
+++ expat-2.2.5/configure.ac
@@ -53,8 +53,6 @@ LIBAGE=6       # CMakeLists.txt!
 CPPFLAGS="${CPPFLAGS} -DHAVE_EXPAT_CONFIG_H"
 AC_CONFIG_HEADER(expat_config.h)
 
-sinclude(conftools/ac_c_bigendian_cross.m4)
-
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
 
